server/src/entity/Section.ts

```diff
@@ -14,7 +14,7 @@ import { Table } from './Table';
 
 @Entity()
 export class Section extends BaseEntity {
-  @PrimaryGeneratedColumn()
+  @PrimaryGeneratedColumn('uuid')
```

Comment:
why do we need uuid for section?

Reply:
you had it. I just added a name...

Reply:
its not natively a uuid, `@PrimaryGeneratedColumn` simply increments a counter. So its the difference between `Seat #1` and `Seat #safdsa-fea4143-fdafe-43214`

Reply:
oh isee